tests/test_fsclient.py

```diff
@@ -117,24 +119,19 @@ def test_all_documents():
 
 test_insert_cif = [({'intensity': [], 'q': [], 'ttheta': [], 'wavelength': 0.111111, '_id': 'ts1129'},
                    {'intensity': [], 'q': [], 'ttheta': [], 'wavelength': 0.111111, '_id': 'ts1129'}),
-                   ('bad_case_test', 'bad_case_test')]
+                   ('bad_case_test', EXEMPLARS['calculated'][-1])]
 @pytest.mark.parametrize('input, result', test_insert_cif)
 def test_insert_one(make_db, rc, input, result):
     client = FileSystemClient(rc)
     client.open()
 
-    actual = client.dbs
-    expected = connect_db(rc)[1]
-    assert actual == expected
-
     dbname = 'local'
     collname = 'calculated'
 
-    try:
-        client.insert_one(dbname, collname, input)
-        assert list(client.dbs[dbname][collname].values())[0] == result
-    except TypeError:
-        print('Input type should be json (dict).')
+    client.load_database(pydr_rc['databases'][0])
+    client.insert_one(dbname, collname, input)
```

Comment:
Please check we have the right logic here.  I think it should look like:

1.  take exemplar and create a filesystem database with exemplar somewhere in tempdir
2.  connect to this db
3.  insert_one into this db
4.  read the file from the tempdir file
5. make sure it contains the original exemplar and additionally the thing that was added.

I don't see the read so I am not sure we are testing that the file was updated.

Reply:
I think since we are calling the `rc` fixture when running the test, the `make_db` and `rc` functions in conftest.py automatically generates a filesystem database with exemplar in tempdir. So if we `print(client.dbs)` after `client.insert_one`, the database contains `{'local': {'calculated': {...EXEMPLAR, INSERTED_DATA}, {'measured': {...EXEMPLAR}}}}`. Also this is what I originally intended to do when I wrote `assert list(client.dbs[dbname][collname].values())[-1] == result`.